### ISSUE TITLE
Missing definition

### ISSUE THREAD

**Issue by MaTribu** [MicrosoftDocs/windows-itpro-docs#3179]
SKU is used without definition.
So, what is SKU ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6d1dee2b-d86a-0019-f689-81483fab31f5
* Version Independent ID: 02722ca2-1dde-aafb-22e3-67259c4ece37
* Content: [Windows Defender Antivirus on Windows Server 2016](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/windows-defender-antivirus-on-windows-server-2016#feedback)
* Content Source: [windows/security/threat-protection/windows-defender-antivirus/windows-defender-antivirus-on-windows-server-2016.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/windows-defender-antivirus/windows-defender-antivirus-on-windows-server-2016.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @andreabichsel
* Microsoft Alias: **v-anbic**

**Comment by illfated** [comment]
I presume you are asking for the definition of SKU (but I could be wrong).
- https://www.investopedia.com/terms/s/stock-keeping-unit-sku.asp :
> #### What Is a Stock Keeping Unit - SKU?
> A stock keeping unit (SKU) is a machine-readable bar code that’s most often seen printed on product labels, that lets vendors expeditiously scan and track the movement of inventory.

- https://en.wikipedia.org/wiki/Stock_keeping_unit :
> In the field of inventory management, a stock keeping unit (SKU /ˌɛsˌkeɪˈjuː, ˈsk(j)uː/) is a distinct type of item for sale,[1] such as a product or service, and all attributes associated with the item type that distinguish it from other item types. For a product, these attributes could include manufacturer, description, material, size, color, packaging, and warranty terms. When a business takes inventory of its stock, it counts the quantity it has of each SKU.

> SKU can also refer to a unique identifier or code that refers to the particular stock keeping unit. These codes are not regulated or standardized. When a company receives items from a vendor, it has a choice of maintaining the vendor's SKU or creating its own. This makes them distinct from Global Trade Item Number (GTIN), which are standard, global, tracking units. Universal Product Code (UPC), International Article Number (EAN), and Australian Product Number (APN) are special cases of GTINs.

In practical terms, the SKU is usually an alphanumeric model name/number of the computer/server machine sold by the manufacturer, distinguishing the different models for sale (and can also be used in inventory lists).

**Comment by mypil** [comment]
@officedocsbot assign @mypil

**Comment by mypil** [comment]
@MaTribu - Thank you for submitting feedback.

From our understanding, the issue has been answered by illfated. If you feel it hasn't been resolved please re-open this issue.

Thank you for engaging with the community here for the docs.

**Comment by MaTribu** [comment]
I think that the answer was done without reading the source.
We are not here "In the field of inventory management".
So no, the issue is not solved.
Please keep this subject open.

**Comment by illfated** [comment]
Could you rephrase your question to make it easier to answer more specifically?
The question, in its current form, is too open to interpretation:
> So, what is SKU ?

**Comment by MaTribu** [comment]
How can I rephrase it ???
First thing is to read and take into account the whole request, which is : "**SKU is used without definition. So, what is SKU ?**"
Which word did you not understand ?
How can you imagine to understand BBB without having the definition of BBB.
No chance !
As the author did not take care to put the definition of acronyms he is using, no one can understand this.
I really don't know how to make it more clear !!!
Use simple words, and give definitions, so that your text is understandable.
Is it so difficult ?

**Comment by illfated** [comment]
Sorry to hear that you are having a hard time understanding what SKU is. I thought you were an IT professional. Most professional IT people who purchase company equipment via retail channels are almost daily exposed to the term "SKU", either in their online ordering forms or via the shipment information, spedition parcel papers or purchase bills forwarded to the accounting department.

The term "SKU" does not need to be complicated, if you simply look at it as a specific way of saying "computer brand model name number tag", but it takes a full sentence or more text to explain what an SKU is for people who don't buy computers as part of their job in the IT department. Are you certain that understanding what SKU is can be counted as a common problem among IT professionals?

**Comment by illfated** [comment]
By the way, in order to put the explanation into context, you could choose to read the sentence as:
> The user interface is installed by default on some specific computer brand model units,
 but is not required.

I think it would be unfair with regards to the intended audience of this article to infer that they are unaware of what SKU means, at least if they work in the IT industry or are in need of using the information provided in the article.

**Comment by mypil** [comment]
@MaTribu - The SKU information shared by @illfated is actually correct. By defintion, SKU represents a purchasable **Stock Keeping Unit (SKU)** under a product.  For more details, please click on this link- **https://docs.microsoft.com/en-us/partner-center/develop/product-resources#sku**

I hope this clarifies and answers your question. Thank you for engaging with the community here for the docs.

**Comment by mypil** [comment]
@officedocsbot close
